command/cli.go

```diff
@@ -98,18 +96,16 @@ func (cli *CLI) Run(args []string) int {
 	return exitCode
 }
 
-func (cli *CLI) groupedHelpFunc(mcli.HelpFunc) mcli.HelpFunc {
-	return func(commands map[string]mcli.CommandFactory) string {
-		c := make(map[string]string)
-		for _, v := range commonCommands {
-			c[v] = printCommand(v, commands[v])
-		}
-
-		return helpFunc(c, "Usage CLI: consul-terraform-sync <command> [-help] [options]\n", nil)
+func help(commands map[string]mcli.CommandFactory) string {
+	c := make(map[string]string)
+	for _, v := range commonCommands {
+		c[v] = generateCommandHelp(v, commands[v])
 	}
+
+	return generateHelp(c, "Usage CLI: consul-terraform-sync <command> [-help] [options]\n", nil)
 }
 
-func helpFunc(commands map[string]string, usage string, omitFlags []string) string {
+func generateHelp(commands map[string]string, usage string, omitFlags []string) string {
```

Comment:
```suggestion
// To support usage by the start command, this function takes a map[string]string and 
// allows for omission of flags. This functionality will not be needed once running CTS 
// without command arguments is deprecated in a future major release.
func generateHelp(commands map[string]string, usage string, omitFlags []string) string {
```

Reply:
to add more context on why the function signature includes a map[string]string instead of the command factory